projects/coordinator/src/test/java/org/batfish/coordinator/WorkMgrTest.java

```diff
@@ -118,4 +118,39 @@ public void listSortedQuestionNames() throws IOException {
     assertThat(questions.size(), is(3));
     assertThat(questions.toString(), equalTo("[access, initinfo, nodes]"));
   }
+
+  @Test
+  public void getEmptyContainer() throws IOException {
+    String containerName = "myContainer";
+    _folder.newFolder(containerName);
+    Main.mainInit(new String[] {"-containerslocation", _folder.getRoot().toString()});
+    Path containerDir = Paths.get(_folder.getRoot().toPath().resolve(containerName).toString());
+    SortedSet<String> containerInfo = _manager.getContainer(containerDir);
```

Comment:
Most of the lines in these tests are identical. Can you pull all the setup of the Main and the containers to a helper function and simplify the tests dramatically?